### PR TITLE
ci(docker): add rosdistro to image names

### PR DESCRIPTION
**Pull request by kenji-miyake**
## Description

To support both Galactic and Humble, I'd like to add `rosdistro` to Docker image names.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by kenji-miyake** [comment]
CI: https://github.com/autowarefoundation/autoware/runs/6547565339?check_suite_focus=true

**Comment by kenji-miyake** [comment]
Worked. https://github.com/autowarefoundation/autoware/runs/6547565339?check_suite_focus=true#step:5:299

```
Docker tags
  ghcr.io/autowarefoundation/autoware-universe:galactic-latest-amd64
  ghcr.io/autowarefoundation/autoware-universe:galactic-20220523-amd64
```

**Comment by kenji-miyake** [comment]
In another PR, I'll add a CI that will create an alias from `galactic-latest` to `latest`, in a similar way to https://github.com/autowarefoundation/autoware/blob/main/.github/actions/update-docker-manifest/action.yaml.

File: docker/build.sh [hunk 1]
```diff
@@ -21,5 +21,5 @@ docker buildx bake --load --progress=plain -f "$SCRIPT_DIR/autoware-universe/doc
     --set "*.args.CUDA_IMAGE_TAG=$cuda_image_tag" \
     --set "*.args.CUDNN_VERSION=$cudnn_version" \
     --set "*.args.TENSORRT_VERSION=$tensorrt_version" \
-    --set "devel.tags=ghcr.io/autowarefoundation/autoware-universe:latest" \
-    --set "prebuilt.tags=ghcr.io/autowarefoundation/autoware-universe:latest-prebuilt"
+    --set "devel.tags=ghcr.io/autowarefoundation/autoware-universe:$rosdistro-latest" \
+    --set "prebuilt.tags=ghcr.io/autowarefoundation/autoware-universe:$rosdistro-latest-prebuilt"
```
Comment (xmfcx) [inline docker/build.sh:25]: Everything seems ok.

For images using these tags, I see this PR:
- https://github.com/autowarefoundation/autoware/pull/335

Are there other PRs planned related to this?

**Review by xmfcx — APPROVED** [review]

File: docker/build.sh (hunk 1 above)
Comment (kenji-miyake) [reply docker/build.sh:25]: > Are there other PRs planned related to this?

Not directly related, but there are some related PRs planned:
- Add humble images
- Add no-CUDA images

Comment (xmfcx) [reply docker/build.sh:25]: I see, thanks.

**Pull request merged**